examples/LAPACK/foo.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream> // std::cout
+#include <cblas.h>
```

Comment:
> Does it work natively on Windows?

There are no working toolchains on Windows in `pkg.openblas`:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1047